### PR TITLE
Pinning golang version for doc builds to 1.21.7

### PR DESCRIPTION
**Pull request by jpodivin**
Fixing the argument

As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:
  - [x] README in the role
  - [x] Content of the docs/source is reflecting the changes

### CODE REVIEW

File: ci/playbooks/pre-doc.yml [hunk 1]
```diff
@@ -23,5 +23,18 @@
         name:
           - make
           - python3
-          - golang
```
Comment (fao89) [inline ci/playbooks/pre-doc.yml:26]:
```suggestion
    - golang-1.21.7-1.el9
```

File: ci/playbooks/pre-doc.yml [hunk 2]
```diff
@@ -22,6 +22,6 @@
       ansible.builtin.package:
         name:
           - make
-          - python3
-          - golang
+          - python3a
```
Comment (rebtoor) [inline ci/playbooks/pre-doc.yml:25]: typo?

Comment (jpodivin) [reply ci/playbooks/pre-doc.yml:25]: yeah.

File: ci/playbooks/pre-doc.yml (hunk 1 above)
Comment (jpodivin) [reply ci/playbooks/pre-doc.yml:26]: It won't last, as the repo won't carry this version forever. But other options are not workable either.

**Review by fao89 — APPROVED** [review]

**Pull request merged**